src/variable.cpp

```diff
@@ -123,10 +123,13 @@ bool Variable::addEquivalence(const VariablePtr &variable1, const VariablePtr &v
 
 bool Variable::removeEquivalence(const VariablePtr &variable1, const VariablePtr &variable2)
 {
-    bool equivalence_1 = variable1 != nullptr ? variable1->pFunc()->unsetEquivalentTo(variable2) : false;
-    bool equivalence_2 = variable2 != nullptr ? variable2->pFunc()->unsetEquivalentTo(variable1) : false;
+    if (variable1 != nullptr && variable2 != nullptr) {
```

Comment:
For readability: `if ((variable1 != nullptr) && (variable2 != nullptr)) {`. The same in other parts of this PR.

Reply:
[c5a9075](https://github.com/cellml/libcellml/pull/1122/commits/c5a9075d621775d0c9e8fdf7bd80f1c870090431)